README.md

```diff
@@ -50,3 +50,14 @@ You should see output similar to the following:
  * Debugger PIN: 226-556-590
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+## Trello and the API Key and Tokens
+
+We're going to be using Trello's API to fetch
+and save to-do tasks. In order to call their
+API, you need to first create an account https://trello.com/signup , then
+generate an API key and token by following
+the instructions here. https://trello.com/app-key 
+Add these to the .env file and ensure this file is in the gitignore file so this does not get committed. 
+
+You also need to get the list ids for the todo, doing and done lists by clicking on a card in each of these lists in trello , then click on Share and then Export JSON. Within the JSON response you can find the idList example - "idList":"6047ecb966a18d23e8029cd2". Add these ids into the .env file too. 
```

Comment:
Nice instructions, but it would be good to make this even clearer by providing the relevant keys, e.g. we could add to the `.env.template` file:
```
# Trello variables
API_KEY=
TOKEN_KEY=
TODO_LISTID=
DOING_LISTID=
DONE_LISTID=
```